projects/packages/status/tests/php/test-modules.php

```diff
@@ -0,0 +1,232 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Tests for Automattic\Jetpack\Status\Modules methods
+ *
+ * @package automattic/jetpack-status
+ */
+
+namespace Automattic\Jetpack;
+
+use Brain\Monkey;
+use Brain\Monkey\Functions;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Status test suite.
+ */
+class Test_Modules extends TestCase {
+
+	/**
+	 * Test setup.
+	 *
+	 * @before
+	 */
+	public function set_up() {
+		parent::setUp();
+		Monkey\setUp();
+
+		// Resetting the static property to avoid using PHPUnit's serialization of properties.
+		$reflection = new \ReflectionProperty( '\Automattic\Jetpack\Modules', 'enforced_modules' );
+		$reflection->setAccessible( true );
+		$reflection->setValue( array() );
+
+		Functions\when( 'is_multisite' )->justReturn( false );
+	}
+
+	/**
+	 * Test teardown.
+	 *
+	 * @after
+	 */
+	public function tear_down() {
+		parent::tearDown();
+
+		$container = \Mockery::getContainer();
+		if ( $container ) {
+			$this->addToAssertionCount(
+				$container->mockery_getExpectationCount()
+			);
+		}
+
+		Monkey\tearDown();
+		\Mockery::close();
+	}
+
+	/**
+	 * Tests that there are no enforced modules by default
+	 */
+	public function test_no_enforced_modules() {
+		Functions\when( 'get_option' )->justReturn( array() );
```

Comment:
Probably it's a dumb question, but is this test actually testing anything? From my understanding, we mock `get_option` response to return an empty array and then checking if it is basically true. So if default modules were set _before_ `Jetpack_Options::get_option`, this test won't fail.

Reply:
Yes, I should have used the actual filter here too, good point. In recent events this PR will probably get saved for later, but still, this is a good suggestion for improvement, thanks!
p9dueE-60f-p2